docs/connectors/slack.md

```diff
@@ -2,51 +2,88 @@
+* Navigate to "OAuth Tokens & Redirect URLs" and click the "Install to Workspace" button. 
+* Take note of the "Bot User OAuth Access Token" as this will be the `bot-token` you need for your configuration (the bot-token will start with `xoxb-`).
 
 ## Configuration
 
 ```yaml
 connectors:
   slack:
     # required
-    token: "xoxb-abdcefghi-12345"
+    bot-token: "xoxb-abdcefghi-12345"
     # optional
-    user-token: "xoxp-abcdef-123" # default None. ** 
-    bot-name: "mybot" # default "opsdroid"
+    socket-mode: true # defaul true. *
+    app-token: "xapp-abdcfkje-12345" # socket-mode needs to be true
+    bot-name: "mybot" # default "opsdroid" **
+    icon-emoji: ":smile:" # default ":robot_face:" **
     default-room: "#random" # default "#general"
-    icon-emoji: ":smile:" # default ":robot_face:"
-    connect-timeout: 10 # default 10 seconds
-    chat-as-user: true # default false
     start_thread: false # default false. if true, opsdroid will start a thread when replying to a message
 ```
-** `user-token` has to be added if you want to send one of the following Events: NewRooom, RoomName, JoinRoom, UserInvite, RoomDescription. From the **classic app** the `user-token` will be the token starts with: `xoxp-`. Note that for you will have to add the scopes in your Slack App to use the different type of events.
 
+\* when `socket-mode` is true, you need to set also an `app-token` (more info: [app level tokens](https://api.slack.com/authentication/token-types#app)
+** In order for `bot-name` and/or `icon-emoji` to work, the `chat:write.customize` scope will have to be selected
+
+### Choose the Backend API
+
+You need to choose between two backends. The [Events API](https://api.slack.com/apis/connections/events-api) or [Socket Mode](https://api.slack.com/apis/connections/socket).
+
+If you are unsure which one is the best for you, [Slack Faq](https://api.slack.com/faq#events_api) provide differences between those two.
+
+**Socket Mode**
+* Go to your [Slack App](https://api.slack.com/apps)
+* On the left columnt go to "Socket Mode" and toogle the "Enable Socket Mode"
+* Copy your new token add add it to your opsdroid configuration file as your `app-token`. (the app-token will start with xapp-abcdef-1233)
+
+**Events API**
+* Make sure to set `socket-mode` to `false`
+* You will need an **endpoint** which exposes your Opsdroid to the **internet**. [Exposing Opsdroid via tunnels](https://docs.opsdroid.dev/en/latest/exposing.html) might help out.
+* Go to your [Slack App](https://api.slack.com/apps)
+* On the left column go to "Event Subscriptions" and toogle the "Enable Events"
+* Under "Request URL" add the `/connector/slack` uri to your endpoint: https://slackbot.example.com/connector/slack. Note that you will have to have your Opsdroid instance running so Slack can validate the endpoint.
+
+### Subscribe to events
```

Comment:
Is this part of the Events API section? 

Could you user headers instead of bold text here so the hierarchy is correct.

Reply:
I think this might be the problem why you are not getting a response from opsdroid. This section belongs to the events API **and** to the socket mode. Meaning you have to subscribe to the events you want to receive from slack regardless of the backend you are using.

If want opsdroid to receive messages that go into a public channel or a dm to opsdroid you will need to subscribe to these events:

![image](https://user-images.githubusercontent.com/9044615/111585945-60a79c80-87c0-11eb-8ea1-a2ff8b25ed5e.png)

If this was your problem, then it means documentation was a bit hard to follow and might need some tweaking.  Suggestions are welcome :)

Reply:
@jacobtomlinson gentle ping, just curious if you were able to test again by adding the events

Reply:
Sorry I haven't managed to yet. Time is a little tight for me at the moment. Hoping to get to this over the weekend.